internal/ingress/controller/template/configmap.go

```diff
@@ -171,6 +173,11 @@ func ReadConfig(src map[string]string) config.Configuration {
 		skipUrls = splitAndTrimSpace(val, ",")
 	}
 
+	if val, ok := conf[skipAccessLogHTTPStatuses]; ok {
```

Comment:
Why not doing this as an array of int as `CustomHTTPErrors: []int{},`?

Reply:
Look at https://github.com/kubernetes/ingress-nginx/blob/main/internal/ingress/controller/template/configmap.go#L152

Unless there is another way of doing it  (like passing 2xx as a valid ignorable code) I would prefer this is also an array of ints

Reply:
If I understand the question, and I may not because I am not proficient in golang, I implemented this as a list of items that would be interpreted as regular expressions because I thought that'd be easier to use than having to explicitly list every status code for which you want to skip logging. Ultimately as long as the end result functions the same and it's not onerous to use I don't really mind, though.

Reply:
Ok, I saw this is being used in a map. I'm just wondering if it is valuable to allow to remove logs from a whole set of codes (using map regex) and not per code and pure ints.

My concern is that we are opening here a chance for people misconfigure ingress-nginx. This is aconfigmap and admin should know what is being done, so I'm less concerned with us allowing arbitrary data here, just asking if we should really keep it wide open now.

Otherwise, just fix my other comment below and lgtm

Thanks

Reply:
I like the idea of leaving this as a regex and agree that there is probably less concern here, than something like an annotation, of someone doing the wrong thing. Additionally, an nginx admin can manually do this today and it's a fully supporte nginx feature, so I am not sure if there's additional reason why we'd need to project people from themselves here, but I am also not a project maintainer. Ultimately I'm fine with whichever way, but it sounds like you're OK leaving it as-is for now, and so am I.